components/NewsAndEvents.tsx

```diff
@@ -0,0 +1,89 @@
+"use client";
+import React from "react";
```

Comment:
```suggestion
import { useState } from "react";
```
And then just use `useState` instead of `React.useState`. This way, you are importing the whole React library.